modules/_node-scripts/preflight/checkPackageJSONFiles.mjs

```diff
@@ -19,31 +18,20 @@ import projectScopeRequire from '../util/projectScopeRequire.mjs';
  *
  * Returns a (possibly empty) array of error messages.
  */
-export async function checkPackageJSONFiles(all) {
-	let packages = [];
-
-	if (all) {
-		packages = await fg('**/package.json', {
-			ignore: [
-				'_node-scripts',
-				'**/build',
-				'**/classes',
-				'**/frontend-js-jquery-web',
-				'**/node_modules',
-				'**/osb-faro-theme',
-				'**/osb-faro-web',
-				'**/sdk',
-				'test',
-			],
-		});
-	}
-	else {
-		const rootDir = await getRootDir();
-
-		packages = (await getGitModifiedFiles())
-			.filter((file) => file.endsWith('package.json'))
-			.map((file) => path.join(rootDir, '..', file));
-	}
+export async function checkPackageJSONFiles() {
```

Comment:
So we just check all package.json every time, even if a PR only makes changes to one or two?

Reply:
yes, because it's so fast it doesn't make sense to make logic more complex